UnityProject/Assets/Scripts/Systems/Inventory/ItemStorage.cs

```diff
@@ -158,13 +160,25 @@ public bool ServerTryTransferFrom(ItemSlot inSlot)
 
 	public bool ServerTryRemove(GameObject InGameObject, bool Destroy = false)
 	{
-		var Item = InGameObject.GetComponent<ItemAttributesV2>();
+		ItemAttributesV2 Item = InGameObject.GetComponent<ItemAttributesV2>();
 		if (Item == null) return false;
-		var slots = GetItemSlots();
+		IEnumerable<ItemSlot> slots = GetItemSlots();
+		HealthV2.BodyPart mobHealth = InGameObject.GetComponent<HealthV2.BodyPart>();
 		foreach (var slot in slots)
 		{
 			if (slot.Item.OrNull()?.gameObject == InGameObject)
 			{
+				if(mobHealth != null)
+				{
+					Debug.Log("health found");
```

Comment:
use logger instead of Debug and try adding the log category.

Reply:
also this particular log seems like is not needed anymore.

Reply:
Ah this is from my tests, gonna remove that now as it's not important to be logged.

Reply:
wait a minute, this is outdated. Are you sure you're looking at the up to date commits?

Reply:
ah you're right, I was reading the particular commit